{"name": "test_add_member_fail[user4--手机号不能为空]", "status": "failed", "statusDetails": {"message": "AssertionError: \nExpected: '手机号不能为空'\n     but: was <None>", "trace": "self = <testcases.test_addMember.TestAddMember object at 0x00000262C82436A0>\nuser = 'user4', phone = '', expect = '手机号不能为空'\n\n    @pytest.mark.parametrize(\"user,phone,expect\", test_data['test_add_member_fail'])\n    def test_add_member_fail(self, user, phone, expect):\n        fail_text = self.addMember.add_member_fail(user, phone)\n>       assert_that(fail_text, equal_to(expect))\nE       AssertionError: \nE       Expected: '手机号不能为空'\nE            but: was <None>\n\ntestcases\\test_addMember.py:29: AssertionError"}, "attachments": [{"name": "log", "source": "eb4df257-b5df-4f6d-b045-94928ba991af-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "user", "value": "'user4'"}, {"name": "phone", "value": "''"}, {"name": "expect", "value": "'手机号不能为空'"}], "start": 1626595347011, "stop": 1626595358236, "uuid": "c41c5cb2-095e-4621-a22d-0d8cb95f9966", "historyId": "3b4d86387144853d51a01719e95b7f7c", "testCaseId": "b251c483add15ec609094c40b1ea12dc", "fullName": "testcases.test_addMember.TestAddMember#test_add_member_fail", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_addMember"}, {"name": "subSuite", "value": "TestAddMember"}, {"name": "host", "value": "z"}, {"name": "thread", "value": "2536-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_addMember"}]}